src/components/cc-orga-member-card/cc-orga-member-card.js

```diff
@@ -106,8 +95,12 @@ export class CcOrgaMemberCard extends LitElement {
     /** @type {Ref<CcSelect>} */
     this._roleRef = createRef();
 
-    /** @type {string} Set by `withResizeObserver` mixin. See the `onResize` method for more info. */
-    this._size = '';
+    /** @type {number|null} set within the `willUpdate` lifecycle hook triggered everytime a resize happens */
+    this._size = null;
+
+    this._componentWidthObserver = new ResizeController(this, {
+      callback: getWidthOnResize,
```

Comment:
This is actually not mandatory.
Since we get the `width` of the component by using `element.getBoundingClientRect()`, we could just set up the controller with no callback and do:

```js
  this._size = this.getBoundingClientRect();
```

within `willUpdate` directly :thinking:.